tests/e2e/utils/killApp.ts

```diff
@@ -1,7 +1,11 @@
 import config from '../config';
 import execAsync from './execAsync';
 
-const killApp = function (platform = 'android', packageName = config.MAIN_APP_PACKAGE): Promise<void> {
+type PromiseWithAbort = Promise<string | void> & {
+    abort?: () => void;
+};
+
+const killApp = function (platform = 'android', packageName = config.MAIN_APP_PACKAGE): PromiseWithAbort {
```

Comment:
Because this function return `execAsync(`adb shell am force-stop ${packageName}`);`

https://github.com/Expensify/App/blob/e9fb49d9f41972a04bf07905b228314a3fdce8d4/tests/e2e/utils/execAsync.ts#L5

should use the same type

Reply:
Updated